woodwork/schema.py

```diff
@@ -426,13 +438,16 @@ def _filter_cols(self, include, col_names=False):
                 tags_used.add(selector)
             # Determine if the selector is a column name
             if col_names and selector in self.columns:
-                cols_to_include.add(selector)
+                match_cols.add(selector)
 
         for col_name, col in self.columns.items():
             if _get_ltype_class(col['logical_type']) in ltypes_used or col['semantic_tags'].intersection(tags_used):
-                cols_to_include.add(col_name)
+                match_cols.add(col_name)
 
-        return list(cols_to_include)
+        if include is not None:
+            return list(match_cols)
+        else:
+            return [col for col in self.columns.keys() if col not in match_cols]
```

Comment:
I know the opposite of the `include` block is going to be the `exclude` logic and that we've already confirmed that there will be either `include` or `exclude` present, but I think it might be nice to do `elif exclude is not None:` to remind anyone reading the code here that exclude is the other behavior here.

Reply:
Sure - make the code a little more explicit which is good. Made that change here and above as well.